packages/ui/src/Radio.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { ChangeEvent, FC, FocusEvent, useRef } from 'react'
+import styles from './Radio.module.scss'
+import classNames from 'classnames'
+import { Help, helpTooltipId } from './Help'
+import { v4 as uuid } from 'uuid'
+
+type RadioCoreProps = {
+  name: string
```

Comment:
`Radio` does not exist outside of `RadioGroup`, right? Moreover, all `Radio` elements inside of the same group have the same name. Couldn't we create a context inside of the `RadioGroup`, and make the `Radio` consume the `name` from the context? It would save us the error of when we supply different names to the radios of the same group by mistake. WDYT?

Reply:
I tried to support it outside, but I'm just trying to make things as similar to native HTML as possible. I should probably stop doing it and start doing it more pragmatic. Context sound great!